src/knex-tables.d.ts

```diff
@@ -131,7 +131,7 @@ declare module "knex/types/tables" {
     monthly_email_optout: boolean;
     monthly_monitor_report_at: null | Date;
     monthly_monitor_report: boolean;
-    breach_resolution: BreachResolution;
+    breach_resolution?: BreachResolution;
```

Comment:
Hmm, what was the reason behind this change? Inspecting my database, this appears to be a non-nullable column?